features/claims/litigator/scheme_nine/transfer_claim_draft_submit.feature

```diff
@@ -15,7 +15,7 @@ Feature: Litigator partially fills out a draft transfer claim, then later edits
     Then I choose the litigator type option 'New'
     And I choose the elected case option 'No'
     And I select the transfer stage 'Before trial transfer'
-    And I enter the transfer date '2015-05-21'
+    And I enter the transfer date '3' years ago
```

Comment:
The quote marks are being picked up in the step definition so `years` is becomes `'3'` and this doesn't convert correctly to an integer.

```suggestion
    And I enter the transfer date 3 years ago
```